### ISSUE TITLE
add picture of editing with puppet-debugger evaulation

### ISSUE THREAD

**Issue by logicminds** [voxpupuli/puppet-mode#81]
I would really like to see this in action.  Can you add a picture that shows the puppet-debugger integration.

**Comment by rski** [comment]
Good idea, it is be blocked by https://github.com/voxpupuli/puppet-mode/issues/82 because i need a working environment to get the screenshots in :)

**Comment by rski** [comment]
@logicminds  something like this?

![2017-03-16-000830_1268x1041_escrotum](https://cloud.githubusercontent.com/assets/2960312/23973007/d8e38c60-09dc-11e7-9bad-f12140f2069d.png)

**Comment by logicminds** [comment]
As a VIM user I am offended.  ;)

Yea this looks good, why there extra newlines on the same row though?

**Comment by rski** [comment]
I like seeing something like this too when evaluating a package/mode:
https://asciinema.org/a/107395

wrt the newlines, probably because it feeds the whole buffer so it also feeds the buffer's newlines. I'll look into it.
